src/components/pages/PageCardsEdit/PageCardsEdit.test.jsx

```diff
@@ -0,0 +1,296 @@
+
+      expect(spyDoc).toBeCalledWith(id);
+    });
+
+    it('should render the card', async () => {
+      const card = cardMock;
+      const { wrapper } = await setup({ card });
+      wrapper.update();
+      const cardEl = findByTestAttr(wrapper, 'page-card-edit__card');
+
+      expect(cardEl.length).toBe(1);
+    });
+
+    it('should default to 0', async () => {
+      const card = null;
+      const { wrapper } = await setup({ card });
+      wrapper.update();
+      const cardEl = findByTestAttr(wrapper, 'page-card-edit__card');
+
+      expect(cardEl.length).toBe(0);
+    });
+
+    describe('Displaying Card Props', () => {
+      it('should display the card title', async () => {
+        const title = 'these are my things';
+        const card = cardMock();
+        card.cardTitle = title;
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        const cardTitleEl = findByTestAttr(wrapper, 'page-cards-edit__title');
+
+        expect(cardTitleEl.prop('value')).toBe(title);
+      });
+
+      it('should display the card type', async () => {
+        const type = 'musings';
+        const card = cardMock();
+        card.type = type;
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        const cardType = findByTestAttr(wrapper, 'page-cards-edit__type');
+
+        expect(cardType.prop('value')).toBe(type);
+      });
+
+      it('should display the card side one', async () => {
+        const sideOne = 'Pyramid Scheme';
+        const card = cardMock();
+        card.side1 = sideOne;
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        const cardSideOne = findByTestAttr(wrapper, 'page-cards-edit__side-one');
+
+        expect(cardSideOne.prop('value')).toBe(sideOne);
+      });
+
+      it('should display the card side two', async () => {
+        const sideTwo = 'Join a Gym';
+        const card = cardMock();
+        card.side2 = sideTwo;
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        const cardSideTwo = findByTestAttr(wrapper, 'page-cards-edit__side-two');
+
+        expect(cardSideTwo.prop('value')).toBe(sideTwo);
+      });
+
+      it('should display the card side three', async () => {
+        const sideThree = 'Take up Whittling';
+        const card = cardMock();
+        card.side3 = sideThree;
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        const cardSideThree = findByTestAttr(wrapper, 'page-cards-edit__side-three');
+
+        expect(cardSideThree.prop('value')).toBe(sideThree);
+      });
+
+      it('should display the card side four', async () => {
+        const sideFour = 'Hitchhike Across Canada';
+        const card = cardMock();
+        card.side4 = sideFour;
+
+        const { wrapper } = await setup({ card });
+        wrapper.update();
+        const cardSideFour = findByTestAttr(wrapper, 'page-cards-edit__side-four');
```

Comment:
So `page-cards-edit__side-four` should really be `p-cards-edit__side-four`. Atomic BEM always uses a shorthand letter for the type (page, atom, molecule, etc).